pkg/traceql/engine.go

```diff
@@ -20,12 +20,18 @@ type Engine struct {
 	spansPerSpanSet int
 }
 
-func NewEngine() *Engine {
+func NewEngine(spss int) *Engine {
 	return &Engine{
-		spansPerSpanSet: 3, // TODO make configurable
+		spansPerSpanSet: spss,
 	}
 }
 
+/*func NewEngineWithSpansPerSpanSet(spansPerSpanSet int) *Engine {
```

Comment:
let's just remove this

Reply:
Very strange. I infact removed it and committed.